scripts/zones/Balgas_Dais/mobs/Wyrm.lua

```diff
@@ -2,7 +2,71 @@
 -- Area: Balga's Dais
 --  Mob: Wyrm
 -- KSNM: Early Bird Catches the Wyrm
+-- For future reference: Trusts are not allowed in this fight 
 -----------------------------------
+require("scripts/globals/status")
+
+function onMobInitialize(mob)
+end
+
+function onMobSpawn(mob)
+    mob:setMobMod(tpz.mobMod.DRAW_IN, 1) -- has a bug during flight, like Tiamat
+    mob:setTP(3000) -- opens fight with a skill
+    mob:delStatusEffect(tpz.effect.TOO_HIGH) -- if wipe during flight and come back with another orb this needs to be cleared
```

Comment:
Is this needed? The status effect _should_ already be gone in a new battlefield, since mobs are destroyed and recreated.

Reply:
The BCNM cleanup should also happen on wipe, that's what you're saying right? I will test this again.

Reply:
bcnm cleanup happens when the bcnm ends for any reason. a wipe and reraise without the bcnm ending will not trigger onMobSpawn again anyway. but when it respawns in a new bcnm, the effect being dleted here will have already been removed by the mob being re-created again.

Reply:
![image](https://user-images.githubusercontent.com/6871475/97819618-b8ad8e80-1c77-11eb-872e-1b2c71cd74d6.png)
 he's not to high at spawn. never too high. :wink: